tests/dm_syncer/conf/dm-task.yaml

```diff
@@ -36,7 +36,16 @@ mysql-instances:
 
 block-allow-list:
   instance:
-    do-dbs: ["dm_syncer"]
+    do-dbs: ["forum_backup_2020", "forum"]
```

Comment:
please keep original "dm_syncer" database, or reuse it to test "block-allow-list"